### ISSUE TITLE
Syntax highlighting for *company-documentation*

### ISSUE THREAD

**Issue by knobo** [anwyn/slime-company#12]
Can we add syntax highlighting for the documentation buffer?

With a minimum of something like this:

``` lisp
(setq slime-company-doc-highlights
      `(("^  [^\\[ ]\\([^:]*\\):" . font-lock-doc-face)))

(define-derived-mode slime-company-doc-mode fundamental-mode "slime-company-doc"
  "Documentation mode for slime-company."
  (setq font-lock-defaults '(slime-company-doc-highlights)))
```

**Comment by knobo** [comment]
I just realised that the mode should be defined in slime, but enabled in *company-documentation* buffer

**Comment by knobo** [comment]
Let's see how this goes.

https://github.com/slime/slime/issues/404

**Comment by anwyn** [comment]
Nag me when this is merged into slime.

**Comment by anwyn** [comment]
Closing this. 
@knobo please submit a new merge request if you have an update.

**Comment by anwyn** [comment]
@knobo I implemented something like this with 7290cbad711a62f76c28e5638d1a4d77197a358c
